ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -21,6 +21,14 @@
 # Some exceptions may exist for the control method (e.g. VAV_ED), and
 # should be avoided if possible.
 
+VAV_SD:
+  guid: "43bff1a1-00b1-4dc6-9aa7-3439b66eb214"
+  description: "Fan powered CAV."
```

Comment:
How is this fan powered? your description doesnt make sense

Reply:
Updated description to reflect the reason for this type.